### PR TITLE
Next Release

### PR DESCRIPTION
**Pull request by mfish33**

### CODE REVIEW

**Comment by AddisonTustin** [comment]
Hrm, beta is out of sync w/ master because of Dependabot commits. Should probably configure so those PRs target beta first...

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -14,7 +14,8 @@ env:
 jobs:
   Integration_And_Deployment:
     runs-on: ubuntu-latest
-
+    env:
```
Comment (AddisonTustin) [inline .github/workflows/deploy.yml:17]: There's probably a [better way we can do this](https://stackoverflow.com/questions/72387730/github-action-how-to-check-if-pr-creator-is-part-of-a-specific-team)? Though, not something I'm _super_ concerned about immediately?

**Review by AddisonTustin — APPROVED** [review]

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (mfish33) [reply .github/workflows/deploy.yml:17]: That was the recommended way since secrets are not available during if statements 🙃

**Comment by mfish33** [comment]
Release train here we go 🎉

**Pull request merged**